src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -697,6 +707,15 @@ def save(self, new_comment=None, notify=True):
       access_limit = self._issue_access_limit
       if access_limit:
         self._data['issueState']['accessLimit'] = {'accessLevel': access_limit}
+      print(list(self.labels))
+      if access_limit == IssueAccessLevel.LIMIT_NONE:
+        # Try to set accessLimit from labels if any.
+        access_limit = _get_access_limit_for_label(limit_view_label)
```

Comment:
hmm, should we always be overriding the access limit when it's specified in a job? it seems a bit confusing to me that this only applies when the default access level is limit_none.

Reply:
I am not sure what the previous behaviour was. I was worried this would allow to open up some restricted issues by mistake. This is why I decided only to set the access level from labels if it's set to None.

Reply:
Alternatively, we could set the custom access level from labels only if it is greater or equal to the current access level limit. WDYT?
Otherwise, we could just override the value always, but we would risk opening up some restricted issues.

Reply:
Thanks for the explanation! What you have seems good to me. We can adjust this later if this ever becomes a problem.